### ISSUE TITLE
Schema::initialize_cross_correlation(): error: feature matching failed (not enough matches)

### ISSUE THREAD

**Issue by lvmingzhi** [dicengine/dice#295]
![image](https://github.com/dicengine/dice/assets/112878439/1551e4c9-8d1e-4d50-abe6-9e695adbb91f)
Is this problem caused by the lack of clear speckle in the shooting

**Comment by dicengine** [comment]
Yes, that's likely the case. You probably need more contrast in the speckle pattern.
